### PR TITLE
Fixes #10185: Lock openjdk to less than 1.8.0.45

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: rubygem-katello.spec [hunk 1]
```diff
@@ -77,6 +77,7 @@ Requires: createrepo >= 0.9.9-18%{?dist}
 Requires: elasticsearch
 Requires: foreman >= 1.7.0
 Requires: java-openjdk >= 1:1.7.0
+Requires: java-openjdk < 1.8.0.45
```
Comment (stbenjam) [inline rubygem-katello.spec]: Why not < 1.8?

Comment (ehelms) [inline rubygem-katello.spec]: It works just fine with 1.8.0.31 which is what has been being pulled in for the last few months.

Comment (stbenjam) [inline rubygem-katello.spec]: Ah ok, thought we needed strictly 1.7.x.

**Comment by stbenjam** [comment]
ACK

**Comment by ehelms** [comment]
I haven't finished testing an actual install. Already found one issue so will likely need to update how the version string is marked.

**Comment by ehelms** [comment]
[test]

**Comment by stbenjam** [comment]
ACK

**Pull request merged**